go.mod

```diff
@@ -1,9 +1,32 @@
 module GoConcurrency-Bootcamp-2022
 
-go 1.16
+go 1.18
```

Comment:
Be careful as changing this compatibility would make others having version 1.16 unable to run your go project

Reply:
Yes, I think the version change 'cause I have installed 1.18 :/ my mistake